pkg/content/content.go

```diff
@@ -133,13 +133,17 @@ func (content *Content) Size() (int64, error) {
 }
 
 // GC clean the local caches by cfg.Provider.GCPolicy configuration
-func (content *Content) GC(ctx context.Context) error {
+func (content *Content) GC(ctx context.Context, threshold ...int64) error {
```

Comment:
Why treat the threshold as a slice here?

Reply:
```scheduled gc task``` should clear more blobs, so ```GC``` should have different thresholds.

Reply:
We can change it to ```threshold int64```